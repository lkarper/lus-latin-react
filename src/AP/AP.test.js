import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import AP from './AP';

describe('AP component', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<AP />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    it('renders the UI as expected', () => {
        const wrapper = shallow(<AP/>);
        expect(toJSON(wrapper)).toMatchSnapshot();
    });
});