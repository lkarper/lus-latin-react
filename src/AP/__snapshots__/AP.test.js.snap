// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AP component renders the UI as expected 1`] = `
<main>
  <h2>
    AP Latin
  </h2>
  <section>
    <h3>
      Deserunt proident fugiat reprehenderit irure et ea nisi ea.
    </h3>
    <p>
      Eu incididunt voluptate occaecat do enim nulla nisi ut commodo ex. Irure aliquip est ut id. Amet anim incididunt minim nisi labore qui id tempor consequat exercitation magna laboris ex labore. Velit nisi elit nulla fugiat.
    </p>
  </section>
</main>
`;
